api/consent-openapi.yaml

```diff
@@ -1917,9 +1917,9 @@ components:
           description: "This could be an FK, but for now we do not have a mapping of identity providers. IDBB may have more requirements."
 
 
-    Agreement:
```

Comment:
This is an actual method name change, right? Make sure to highlight this to @akmad in the release notes as it's a "breaking change" compared to what was done in 1.0

(It's a good change - just needs highlighting)

Reply:
Correct - I would include such changes in the same PR, but there was a preference to keep the Changelog on Confluence. I still prefer having the Changelog maintained as .md in the repo.

Anyways, because of the separate Changelog, I try to write notes about changes in the PR description so I don't end up forgetting things.

https://govstack-global.atlassian.net/wiki/spaces/GH/pages/283115524/Changelog+-+Consent+Building+Block

Everything in this PR is a breaking change :upside_down_face: